docs/provider/vmong5k.rst

```diff
@@ -0,0 +1,105 @@
+.. _vmong5k:
+
+Virtual Machines on Grid'5000
+=============================
+
+It is possible to deploy virtual machines on top of bare nodes of Grid'5000.
+This hybrid approach is useful to take advantage of all resouces available in
+each node. In this way several virtual machines with different roles can be
```

Comment:
Actually, in the current implementation vms with different won't be colocalized (enoslib limitation). But several vms with the same role can coexist.